main.tf

```
@@ -64,8 +64,9 @@ data "aws_cloudwatch_event_bus" "this" {
 resource "aws_cloudwatch_event_bus" "this" {
   count = var.create && var.create_bus ? 1 : 0
 
-  name              = var.bus_name
-  event_source_name = try(var.event_source_name, null)
+  name               = var.bus_name
+  event_source_name  = try(var.event_source_name, null)
+  kms_key_identifier = try(var.kms_key_identifier, null)
```

this line and the line above aren't doing what is intended - it should just be:
```suggestion
  kms_key_identifier = var.kms_key_identifier
```

if you could fix line 68 as well, that would be great!